layouts/partials/sections/home.html

```diff
@@ -28,7 +28,7 @@
 <div class="container-fluid home" id="home">
   <div
     class="background container-fluid"
-    style="background-image: url('{{ strings.TrimSuffix "/" site.BaseURL }}{{ $backgroundImage | relURL }}');"
+    style="background-image: url('{{ $backgroundImage | relURL }}');"
```

Comment:
This doesn't work on https://themes.gohugo.io/. That's why I had to keep it this way.

Reply:
same with base URL set it's strange. how do you set BaseURL on my it's like that:
``baseURL : "https://www.ducamps.win/welcome"``

Reply:
@vincentDcmps I am afraid I didn't understand your comment properly. Can you explain little bit?  Did you mean it doesn't work with `baseURL : "https://www.ducamps.win/welcome"`?

Reply:
With this baseurl it s work only with removing background-image: 
''url('{{ strings.TrimSuffix "/" site.BaseURL }}'' 
It s weird to put this only for background ans not for other logo

Reply:
to reproduce but in config.yaml 
```baseURL: 'https://www.ducamps.win/welcome```
and launch test server with ```hugo server -b localhost/welcome -w```